osu.Game/Screens/Play/HUD/ArgonSongProgressBar.cs

```diff
@@ -173,5 +191,7 @@ public ColourInfo AccentColour
                 set => fill.Colour = value;
             }
         }
+
+        private string formatTime(TimeSpan timeSpan) => $"{(timeSpan < TimeSpan.Zero ? "-" : "")}{Math.Floor(timeSpan.Duration().TotalMinutes)}:{timeSpan.Duration().Seconds:D2}";
```

Comment:
Similarly here I have no idea why you're rolling your own timespan formatting instead of one of [these](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-timespan-format-strings) or [these](https://learn.microsoft.com/en-us/dotnet/standard/base-types/custom-timespan-format-strings).

Reply:
I happened to find this in the `SongProgress` class, since I didn't know another way to implement this.
Given the knowledge base articles, I'll give it a shot then 😋

Reply:
The timespan formatting mentioned above doesn't mean much in this situation, where we are merging hours into minutes (1h -> 60mins). For now I'll still use the existing method...